ee/vellum_ee/workflows/display/vellum.py

```diff
@@ -41,10 +41,6 @@ class NodeDisplayData(UniversalBaseModel):
     comment: Optional[NodeDisplayComment] = None
 
 
-class GenericNodeDisplayData(UniversalBaseModel):
-    position: NodeDisplayPosition = Field(default_factory=NodeDisplayPosition)
```

Comment:
@peicodes any insight to share on why we created a separate Display data class?

Reply:
I think the legacy ones had fields that generic nodes did not, but I don't remember which fields